### PR TITLE
Feature/25

### PR DESCRIPTION
**Pull request by Remmo1**
Basic GRPC functions:
* Generator
* Server
* Client

### CODE REVIEW

File: backend/clap/pom.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.1.4</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.example</groupId>
```
Comment (gcfbn) [inline backend/clap/pom.xml:11]: Maybe we can already use some "real" name? `com.ppea` or something ZPI-related?
It applies to the child artifacts too.

File: backend/clap/.gitignore [hunk 2]
```diff
@@ -0,0 +1,33 @@
+HELP.md
```
Comment (gcfbn) [inline backend/clap/.gitignore:1]: I think we agreed to have a single .gitignore file in the root directory

**Review by gcfbn — CHANGES_REQUESTED** [review]
Aren't we missing the Firebase integration, which is a part of this ticket as well?

File: backend/clap/pom.xml (hunk 1 above)
Comment (Veermove) [reply backend/clap/pom.xml:11]: we can even do `zpi.ppea`

**Comment by Remmo1** [comment]
> Aren't we missing the Firebase integration, which is a part of this ticket as well?

In progress, I am not sure it will be ready today.

File: backend/clap/pom.xml (hunk 1 above)
Comment (Remmo1) [reply backend/clap/pom.xml:11]: ok

File: backend/clap/interface-grpc/src/main/proto/hello.proto [hunk 3]
```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package net.devh.boot.grpc.example;
```
[the remark this answers is not in the document]
Comment (Remmo1) [reply backend/clap/interface-grpc/src/main/proto/hello.proto:3]: I do agree, proto files should be in one place. We need to confirm names and packages. I will do this in the next commit.

**Comment by Remmo1** [comment]
ok it's now much better.

**Review by gcfbn — APPROVED** [review]

**Review by Pandorianim — APPROVED** [review]
Looking good

**Review by Veermove — APPROVED** [review]

**Pull request merged**